### ISSUE TITLE
Update Turkish translations

### ISSUE THREAD

**Issue by eminboydak** [squidfunk/mkdocs-material#5399]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "tr",
  "action.edit": "Düzenle",
  "action.skip": "Ana içeriğe geç",
  "action.view": "Sayfanın kaynağını görüntüle ⬅️",
  "announce.dismiss": "Bir daha gösterme ⬅️",
  "blog.archive": "Arşiv ⬅️",
  "blog.categories": "Kategoriler ⬅️",
  "blog.categories.in": "in ⬅️",
  "blog.continue": "Okumaya devam et ⬅️",
  "blog.draft": "Taslak ⬅️",
  "blog.index": "Dizine geri dön ⬅️",
  "blog.meta": "Metadata ⬅️",
  "blog.references": "İlgili bağlantılar ⬅️",
  "clipboard.copy": "Kopyala",
  "clipboard.copied": "Kopyalandı",
  "consent.accept": "Kabul et ⬅️",
  "consent.manage": "Ayarları yönet ⬅️",
  "consent.reject": "Reddet ⬅️",
  "footer": "Altbilgi ⬅️",
  "footer.next": "Sonraki",
  "footer.previous": "Önceki",
  "header": "Başlık ⬅️",
  "meta.comments": "Yorumlar",
  "meta.source": "Kaynak",
  "nav": "Navigasyon ⬅️",
  "readtime.one": "1 dakika okuma ⬅️",
  "readtime.other": "# dakika okuma ⬅️",
  "rss.created": "RSS beslemesi ⬅️",
  "rss.updated": "Güncellenmiş içeriğin RSS beslemesi ⬅️",
  "search": "Ara ⬅️",
  "search.placeholder": "Ara",
  "search.share": "Paylaş ⬅️",
  "search.reset": "Temizle ⬅️",
  "search.result.initializer": "Arama başlatılıyor ⬅️",
  "search.result.placeholder": "Aramaya başlamak için yazın",
  "search.result.none": "Eşleşen doküman bulunamadı",
  "search.result.one": "1 doküman bulundu",
  "search.result.other": "# doküman bulundu",
  "search.result.more.one": "Bu sayfada 1 tane daha ⬅️",
  "search.result.more.other": "Bu sayfada # tane daha ⬅️",
  "search.result.term.missing": "Eksik ⬅️",
  "select.language": "Dil seç ⬅️",
  "select.version": "Versiyon seç ⬅️",
  "source": "Depoya git",
  "source.file.contributors": "Katkıda bulunanlar ⬅️",
  "source.file.date.created": "Oluşturuldu",
  "source.file.date.updated": "Son Güncelleme",
  "tabs": "Sekmeler ⬅️",
  "toc": "İçindekiler",
  "top": "Başa dön ⬅️"
}[key] }}{% endmacro %}
```

### Before submitting

- [x] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [X] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by squidfunk** [comment]
Awesome, thanks! Added in c0cdb3ca1.

**Comment by squidfunk** [comment]
Ah, I saw to late that you wanted to open a PR yourself – sorry for that! However, I tagged you as a co-author in the commit, so your work is linked to your account – same result effectively ☺️

**Comment by squidfunk** [comment]
Released as part of [9.1.7](https://github.com/squidfunk/mkdocs-material/releases/tag/9.1.7).
